web/ui/react-app/src/contexts/ClipBoardContext.tsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+const ClipboardContext = React.createContext((msg: string) => {
```

Comment:
I would tend to not call this `ClipboardContext`, but just `ToastContext`, to both clarify what it's actually about and make it more general in use. When I first read the name, I assumed this was for storing the clipboard message (but I think keeping the clipboard message just in the actual clipboard as you do it now is good).